rxandroidaudio/src/main/java/com/github/piasy/rxandroidaudio/RxAudioPlayer.java

```diff
@@ -131,11 +135,75 @@ public void call(final SingleSubscriber<? super Boolean> singleSubscriber) {
                 }
             });
         } else {
-            return Single.error(new IllegalArgumentException(""));
+            return Observable.error(new IllegalArgumentException(""));
+        }
+    }
+
+    /**
+     * prepare audio from local file. should be scheduled in IO thread.
+     */
+    public Observable<Boolean> prepare(@NonNull final PlayConfig config) {
```

Comment:
After we get prepared via `prepare`, when could we play it? and when would the returned `Observable` complete?